ui/pages/confirmations/confirmation/templates/create-snap-account.js

```diff
@@ -20,6 +20,11 @@ function getValues(pendingApproval, t, actions, _history, _data, contexts) {
     });
   };
 
+  const onCancel = () => {
+    trackSnapAccountEvent(MetaMetricsEventName.AddSnapAccountCanceled);
+    actions.resolvePendingApproval(pendingApproval.id, false);
+  };
+
```

Comment:
Why did you have to extract this ?

Reply:
In order to avoid code duplication, because this needs to be returned in two places. Any other idea without complicating this too much 😅 ?

Reply:
Oh okay, it's also passed to the props. Okay for now :)